content/js/record.md

```diff
@@ -1,335 +1,189 @@
-  `noteID` as well.
-- You can migrate any integer to auto-incrementing sequence.
-- Our JIT schema at development will migrate the DB schema to sequence. All
-  `noteID` at `Note` will be a sequence type once migrated.
-
-If you wish to override sequence manually, you can do that as well. If the
-provided `noteID` is taken by another record, there will be an error; otherwise,
-the record will be saved and the maximum `noteID` plus one will be used for the
-next record.
-
-``` javascript
-const note = new Note({
-  content: 'Hello World'
-});
-note.noteID = 43;
-skygear.publicDB.save(note);
-```
-
-<a name="reserved"></a>
-### Reserved Columns
-
-There are quite a few reserved columns for storing records into the database.
-The column names are written as **snake_case** while the JS object attributes
-are mapped with **camelCase**. Please notice this one-to-one mapping. When you want
-to query on reserved columns, make sure to use **snake_case**; when you get records
-back as a JS object, make sure to access attributes with **camelCase**. When
-creating and saving records, please avoid using attribute that is the same
-as any one of the camelCase attribute names listed below.
-
-Column Name | Object Attribute | Description
---- | --- | ---
-`_created_at` | `createdAt` | date object of when record is created
-`_updated_at` | `updatedAt` | date object of when record is updated last time
-`_created_by` | `createdBy` | user id of record creator
-`_updated_by` | `updatedBy` | user id of last record updater
-`_owner_id` | `ownerID` | user id of owner
-**N/A** | `id` | record type and record id
-`_id` | `_id` | record id
-
-One quick example:
-
-``` javascript
-skygear.publicDB.query(new skygear.Query(Note))
-  .then((records) => console.log(records[0]));
-```
-
-``` javascript
-/* Type: RecordCls */ {
-  createdAt: new Date("Thu Jul 07 2016 12:12:42 GMT+0800 (CST)"),
-  updatedAt: new Date("Thu Jul 07 2016 12:42:17 GMT+0800 (CST)"),
-  createdBy: "118e0217-ffda-49b4-8564-c6c9573259bb",
-  updatedBy: "118e0217-ffda-49b4-8564-c6c9573259bb",
-  ownerID: "118e0217-ffda-49b4-8564-c6c9573259bb",
-  id: "note/3b9f8f98-f993-4e1d-81c3-a451e483306b",
-  _id: "3b9f8f98-f993-4e1d-81c3-a451e483306b",
-  recordType: "note",
-}
 ```
+Note: the delete operation uses the normal `id` field without underscore.
 
-Query on reserved columns example:
+This API call returns a promise object containing the deleted record or array of deleted records.
 
-``` javascript
-let query = new skygear.Query(Note);
-query.equalTo('_owner', skygear.currentUser.id);
-// '_owner' is an alias for '_owner_id'
-skygear.publicDB.query(query);
+<a name="atomic-save-updates"></a>
+## Atomic Save/Updates
+In a multi-save/update operation, you can make it such that either all or none of the records will be
+saved if an error occurs using the `atomic` option. This call will behave just like a single-item save
+operation:
+```js
+skygear.publicDB.save(arrayOfRecords, { atomic: true });
```

Comment:
Needs example on success/error handling to show how it behaves like a single-item save